mava/components/jax/training/losses.py

```diff
@@ -118,17 +120,51 @@ def policy_loss_grad_fn(
                 # the case of non-shared weights.
                 def policy_loss_fn(
                     policy_params: Any,
+                    policy_states: Any,
                     observations: Any,
                     actions: jnp.ndarray,
                     behaviour_log_probs: jnp.ndarray,
                     advantages: jnp.ndarray,
                 ) -> Tuple[jnp.ndarray, Dict[str, jnp.ndarray]]:
                     """Inner policy loss function: see outer function for parameters."""
-                    distribution_params = network.policy_network.apply(
-                        policy_params, observations
-                    )
+
+                    # TODO (dries): Can we implement something more general here? Like a function call?
+                    if policy_states:
+                        # Recurrent actor.
+                        batch_size = trainer.store.sample_batch_size
+                        seq_len = trainer.store.sequence_length - 1
+
+                        bs_observations = observations.reshape(batch_size, seq_len, -1)
```

Comment:
maybe we can name this batch_seq_observtions

Reply:
I see you did not like my bs name haha. I updated the code.

Reply:
haha :wink: 
Can you please do the same for `bs_policy_states`

Reply:
Updated :)